### PR TITLE
doc: Add docker-compose resources

### PR DESCRIPTION
**Pull request by YazzyYaz**
Hi @jbowen93 please review this PR

### CODE REVIEW

File: docs/nodes/devops-resources.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# DevOps Resources
+
+This section will be provide an overview of different available resources for Node Operators looking to test out Celestia architecture.
+
+## Docker Compose
+
+* One resource for playing with docker-compose for Celestia infrastructure is the [ephemeral-cluster](https://github.com/celestiaorg/ephemeral-cluster) repository. This repository provides infra-as-code specifications on running a docker-compose instance of Celestia devnet. It is not recommended to be used as-is for running nodes individually but can be used as a reference specification for anyone looking to modify it for their own purposes.
```
Comment (jbowen93) [inline docs/nodes/devops-resources.md:7]: Can we replace `docker-compose` with `Docker Compose` in the two places it's used here. The `docker-compose` binary uses the older version (v1) by default which doesn't work with the ephemeral-cluster.

Comment (jbowen93) [inline docs/nodes/devops-resources.md:7]: I'm not sure we want to refer to the ephemeral cluster as a "reference specification". 
It's not targeted at node operators, it's a place to do integration testing for development.

Comment (YazzyYaz) [reply docs/nodes/devops-resources.md]: Check the new updated writing.

Comment (YazzyYaz) [reply docs/nodes/devops-resources.md]: Modified

**Comment by YazzyYaz** [comment]
@jbowen93 please let me know if it's good to go now

**Review by jbowen93 — APPROVED** [review]

**Pull request merged**